### ISSUE TITLE
CSS found in `.mjs` files is not bundled with `unstable_cssSideEffectImports`

### ISSUE THREAD

**Issue by tchak** [remix-run/remix#5562]
### What version of Remix are you using?

1.13.0

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

Install `react-spectrum 3.25.0` and configure `unstable_cssSideEffectImports`. In `3.25.0` they migrated to `.mjs` files.

The problem sims to be here https://github.com/remix-run/remix/blob/f5236c1d880da51d70a8ac24a0922d2e5ded7416/packages/remix-dev/compiler/plugins/cssSideEffectImportsPlugin.ts#L24-L25

### Expected Behavior

CSS found in `.mjs` files should be bundled

### Actual Behavior

CSS found in `.mjs` files is not bundled

**Comment by tchak** [comment]
This patch solves the problem:

```patch
diff --git a/dist/compiler/plugins/cssSideEffectImportsPlugin.js b/dist/compiler/plugins/cssSideEffectImportsPlugin.js
index 791768d7f8625c286ece62f860dcb9c320ed8cbb..bf833305ec40a37e56981ad52b1423395a16bef7 100644
--- a/dist/compiler/plugins/cssSideEffectImportsPlugin.js
+++ b/dist/compiler/plugins/cssSideEffectImportsPlugin.js
@@ -35,13 +35,15 @@ const cssSideEffectFilter = new RegExp(`\\.css${cssSideEffectSuffix.replace("?",
 function isCssSideEffectImportPath(path) {
   return cssSideEffectFilter.test(path);
 }
-const loaders = ["js", "jsx", "ts", "tsx"];
+const loaders = ["js", "jsx", "ts", "tsx", "mjs", "mjsx"];
 const allJsFilesFilter = new RegExp(`\\.(${loaders.join("|")})$`);
 const loaderForExtension = {
   ".js": "js",
   ".jsx": "jsx",
   ".ts": "ts",
-  ".tsx": "tsx"
+  ".tsx": "tsx",
+  ".mjs": "js",
+  ".mjsx": "jsx"
 };
 
 /**
```

**Comment by markdalgleish** [comment]
This was fixed in v1.15.0.
